elements/lisk-validator/src/formats.ts

```diff
@@ -125,3 +127,41 @@ const camelCaseRegex = /^[a-z]+((\d)|([A-Z0-9][a-zA-Z0-9]+))*([a-z0-9A-Z])?$/;
 
 export const camelCase = (data: string): boolean =>
 	camelCaseRegex.exec(data) !== null;
+
+export const version = (data: string): boolean =>
+	/^([0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3})(-(alpha|beta|rc)\.[0-9]{1,3}(\.[0-9]{1,3})?)?$/.test(
```

Comment:
I suggest this regular expression should be flexible and just follow the semantic versioning standard. As the version of the application for any custom application can be different and they can have any prefix than alpha, beta, rc.

Reply:
these regex was there in the application validator, and just moved to the common validator.
Let's create another issue to update the regex

Reply:
https://github.com/LiskHQ/lisk-sdk/issues/5416